projects/view/client/rtProject/rtProjectProfile.css

```diff
@@ -0,0 +1,70 @@
+/*  COLUMN SETUP  */
+.rtProjectProfile .col {
+	display: block;
+	float:left;
+	margin: 1% 0 1% 1%;
+}
+.rtProjectProfile .col:first-child { margin-left: 0; }
+
+/*  GROUPING  */
+.rtProjectProfile .group:before,
+.rtProjectProfile .group:after { content:""; display:table; }
+.rtProjectProfile .group:after { clear:both;}
+.rtProjectProfile .group { zoom:1; /* For IE 6/7 */ }
+
+/*  GRID OF TWELVE  */
```

Comment:
All of these rules are redundant. Please use [Bootstrap grid classes](http://getbootstrap.com/css/#grid) and remove these definitions.